### PR TITLE
Masterofdisaster

### PR DESCRIPTION
**Pull request by marcinpilat**
First pull request:)

### CODE REVIEW

File: src/java/exception/MaxLoadExceededException.java [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package pack;
+package java.exception;
```
Comment (marcinpilat) [inline src/java/exception/MaxLoadExceededException.java]: Root package called `java` isn't best idea. Your root package could be `markowlodarko.trip` so it gives some context.

File: src/java/exception/MaxPassengerExceededException.java [hunk 2]
```diff
@@ -9,6 +9,7 @@
     public MaxPassengerExceededException(int maxPassengers, int expectedPassengersLoad) {
         this.maxPassengers = maxPassengers;
         this.expectedPassengersLoad = expectedPassengersLoad;
+        System.out.println("Za dużo pasażerów! Maksymalna ilość miejsc wynosi "+ getMaxPassengers());
```
Comment (marcinpilat) [inline src/java/exception/MaxPassengerExceededException.java]: That's very bad idea. Exception class is just to keep exception data, not to handle exception (by printing it you're kind'of handling them). This should be in catch block

File: src/java/exception/MaxLoadExceededException.java [hunk 3]
```diff
@@ -9,6 +9,7 @@
     public MaxLoadExceededException(double maxLoad, double expectedLoad) {
         this.maxLoad = maxLoad;
         this.expectedLoad = expectedLoad;
+        System.out.println("Za duże obciążenie! Maksymalne obciążenie wynosi "+getMaxLoad()+ ", a oczekiwana waga to " + getExpectedLoad());
```
Comment (marcinpilat) [inline src/java/exception/MaxLoadExceededException.java]: Same as in `MaxPassengerExceededException .java`

File: src/java/pack/Trip.java [hunk 4]
```diff
@@ -1,81 +1,85 @@
...
+            sum += voyager.getOverallWeight();
+        }
+        return sum;
+    }
+
+    public double getDistance() {
+        return distance;
+    }
+
+    public void doOverallCalculation(Trip trip){
```
Comment (marcinpilat) [inline src/java/pack/Trip.java]: This and `doCalculationsPerPassenger` shouldn't be in this class.

File: src/java/exception/MaxPassengerExceededException.java (hunk 2 above)
Comment (MarkoWlodarko) [inline src/java/exception/MaxPassengerExceededException.java]: Why ?

Comment (MarkoWlodarko) [reply src/java/exception/MaxPassengerExceededException.java]: Is it some common rule or it might cause some errors ?

Comment (marcinpilat) [reply src/java/exception/MaxPassengerExceededException.java]: It won't cause any errors. Check [documentation](https://docs.oracle.com/javase/8/docs/api/java/lang/Exception.html) of `Exception` class. 

> The class Exception and its subclasses are a form of Throwable that indicates **conditions that a reasonable application might want to catch.** 

There is nothing about handling, only **conditions to catch**.

File: src/java/pack/Trip.java (hunk 4 above)
Comment (MarkoWlodarko) [reply src/java/pack/Trip.java]: hmmm. Do you have some suggestions where should I put it ?
Also why these methods shouldn't be in this class ? These were located there in the first place.

**Pull request merged**